doc/releases/migration-guide-3.7.rst

```diff
@@ -45,6 +45,19 @@ Bluetooth HCI
 Controller Area Network (CAN)
 =============================
 
+* Removed the following deprecated CAN controller devicetree properties. Out-of-tree boards using
```

Comment:
Add a link to `can-controller` DTS binding?

Reply:
There is no common binding for CAN controllers, just a base binding included by all CAN controller DTS bindings.